OWNERS

```diff
@@ -3,3 +3,9 @@
 approvers:
   - ahg-g
   - liurupeng
+  - kerthcet
+
+reviewers:
+  - ahg-g
+  - liurupeng
```

Comment:
we may not be able to add Rupeng until he becomes a member

Reply:
Remove from approvers as well, sorry for that Rupeng.